packages/developer-portal/src/components/pages/apps/edit/form-schema/validation-schema.ts

```diff
@@ -115,7 +118,16 @@ export const appEditValidationSchema = object().shape({
         .required(FIELD_REQUIRED)
         .min(150, errorMessages.BETWEEN_MIN_MAX_CHARACTER_LENGTH(150, 1500))
         .max(1500, errorMessages.BETWEEN_MIN_MAX_CHARACTER_LENGTH(150, 1500)),
-      otherwise: string().notRequired(),
+      otherwise: string()
+        .trim()
+        .test({
+          name: 'isValidSummary',
+          message: errorMessages.BETWEEN_MIN_MAX_CHARACTER_LENGTH(50, 150),
+          test: (value) => {
+            if (!value) return true
```

Comment:
If it's an empty string it should pass, because the field is valid if empty but if provided must be between 50 and 150 chars... it's a bit janky I know but tested and works